plugins/cad/src/components/ResourceViewerDialog/components/FirstClassViewerSelector.tsx

```diff
@@ -98,6 +100,13 @@ const getCustomMetadataFn = (
     case 'v1/ConfigMap':
       return getConfigMapStructuredMetadata;
 
+
+    case 'config.porch.kpt.dev/v1alpha1/PackageVariant':
+      return getPackageVariantStructuredMetadata;
+
+    case 'config.porch.kpt.dev/v1alpha2/PackageVariantSet':
+      return getPackageVariantSetStructuredMetadata;
+
```

Comment:
nit: let's move this to above  `case 'fn.kpt.dev/v1alpha1/ApplyReplacements'` to keep this list in alphabetical order with the group version kind.